src/doc/contrib/src/tests/running.md

```diff
@@ -40,6 +40,13 @@ the `CARGO_RUN_BUILD_STD_TESTS=1` environment variable and running `cargo test
 `rust-src` component installed with `rustup component add rust-src
 --toolchain=nightly`.
 
+## Running with `gitoxide` as default git backend
+
+By default, the `git2` backend is used for most git operations.
+With the `CARGO_USE_GITOXIDE_INSTEAD_OF_GIT2=1` environment variable set while building cargo, 
+the `gitoxide` feature will automatically be enabled to use it instead of `git2`
+where possible. This is for use in tests to switch `git` backends without making further adjustments.
```

Comment:
I feel like this should be re-framed to put `CARGO_USE_GITOXIDE_INSTEAD_OF_GIT2` in terms of `-Zgitoxide` to make the relationship  to an unstable feature clear.

Reply:
Thanks for pointing this out. In 339a3efab97dff06623344cb06de961fe5244e2b the relationship was made clearer, along with the motivation for the environment variable.